market_licensing/src/main/java/com/google/android/vending/licensing/PreferenceObfuscator.java

```diff
@@ -42,6 +43,7 @@ public PreferenceObfuscator(SharedPreferences sp, Obfuscator o) {
         mEditor = null;
     }
 
+    @SuppressLint("CommitPrefEdits")
```

Comment:
Do not suppress. Add `mEditor.commit();` after `mEditor = mPreferences.edit();` in if statement

Reply:
Done.